azurestack/resource_arm_loadbalancer_probe.go

```diff
@@ -0,0 +1,281 @@
+package azurestack
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2015-06-15/network"
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurestack/azurestack/utils"
+)
+
+func resourceArmLoadBalancerProbe() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmLoadBalancerProbeCreate,
+		Read:   resourceArmLoadBalancerProbeRead,
+		Update: resourceArmLoadBalancerProbeCreate,
+		Delete: resourceArmLoadBalancerProbeDelete,
+		Importer: &schema.ResourceImporter{
+			State: loadBalancerSubResourceStateImporter,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"resource_group_name": resourceGroupNameSchema(),
+
+			"loadbalancer_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"protocol": {
+				Type:             schema.TypeString,
+				Computed:         true,
+				Optional:         true,
+				StateFunc:        ignoreCaseStateFunc,
+				DiffSuppressFunc: ignoreCaseDiffSuppressFunc,
+			},
+
+			"port": {
+				Type:     schema.TypeInt,
+				Required: true,
+			},
+
+			"request_path": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"interval_in_seconds": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  15,
+			},
```

Comment:
Could we get some validation here too?

`ValidateFunc: validation.IntAtLeast(5),`